src/index.js

```diff
@@ -73,8 +73,8 @@ const component = () => {
     } else {
       todos = JSON.parse(localStorage.getItem('todos'));
     }
-    const m = todos.filter((x) => x.index.toString() !== todo.id);
-    const reset = m.map((e, i) => {
+    const filterArr = todos.filter((x) => x.index.toString() !== todo.id);
+    const reset = filterArr.map((e, i) => {
```

Comment:
- [ ] You "clear completed" method is working is functioning correctly although the functionality for this should be in a different module file that you then import here